MicroAOD/python/MicroAODCustomize.py

```diff
@@ -326,6 +326,37 @@ def customizeDec2016Regression(self,process):
         process.photonMVAValueMapProducer.srcMiniAOD = cms.InputTag("slimmedPhotons")
         process.photonIDValueMapProducer.srcMiniAOD = cms.InputTag("slimmedPhotons")
 
+    # For the whole RunII use the V2 ID, as explained here  https://indico.cern.ch/event/777545/contributions/3234587/attachments/1766655/2868714/micheli_EG_cmsweek_20181206_4.pdf
```

Comment:
How stable is this? Should we foresee to have the ID weights and effective areas file configurable from MetaConditions?

Reply:
if we use this id, it will never change. i copied the structure of the old customize methods. If a new version of the ID will come for ultra-legacy it will have to be updated.